app/components/institution-page/campus-program-application-modal/verify-email-step-container.ts

```diff
@@ -1,15 +1,22 @@
 import Component from '@glimmer/component';
 import type InstitutionMembershipGrantApplicationModel from 'codecrafters-frontend/models/institution-membership-grant-application';
+import { action } from '@ember/object';
 
 interface Signature {
   Element: HTMLDivElement;
 
   Args: {
     application: InstitutionMembershipGrantApplicationModel;
+    onGoBackToEnterEmail: () => void;
```

Comment:
Not really in line with what we normally name args like these, examples: 

<img width="867" height="790" alt="Screenshot 2025-08-16 at 02 43 06" src="https://github.com/user-attachments/assets/c8112537-1e72-47eb-8f88-18ed6c9f5d76" />


Pick something closer to what you see there - the principle is to use terminology that maps more closely to the UI. For example, `onChangeOrResendEmailButtonClick`